packages/core/interfaces/src/index.ts

```diff
@@ -39,11 +39,10 @@ export interface UseUser
   logout: () => Promise<void>;
   changePassword: (
     currentPassword: string,
-    newPassword: string
-  ) => Promise<void>;
+    newPassword: string) => Promise<void>;
+  refreshUser: () => void;
```

Comment:
why it's not promise anymore?
```suggestion
  refreshUser: () => Promise<void>;
```